src/server/views/dynamic/sponsoring/sponsoring.handlebars

```diff
@@ -298,73 +302,162 @@
-                                        {{sponsorings.0.unregisteredSponsor.lastname}}
-                                    {{else}}
-                                        {{#if sponsorings.0.company}}
-                                            {{sponsorings.0.company}} -
-                                        {{/if}}
-                                        {{sponsorings.0.firstname}}
-                                        {{sponsorings.0.lastname}}
-                                    {{/if}}
-                                {{else}}
-                                    {{#if challenges.0.unregisteredSponsor}}
-                                        {{#if challenges.0.unregisteredSponsor.company}}
-                                            {{challenges.0.unregisteredSponsor.company}} -
-                                        {{/if}}
-                                        {{challenges.0.unregisteredSponsor.firstname}}
-                                        {{challenges.0.unregisteredSponsor.lastname}}
-                                    {{else}}
-                                        {{#if challenges.0.company}}
-                                            {{challenges.0.company}} -
-                                        {{/if}}
-                                        {{challenges.0.firstname}}
-                                        {{challenges.0.lastname}}
-                                    {{/if}}
-                                {{/if}}
-                            {{/if}}
-
-                            {{#each members}}
-                                {{firstname}} {{lastname}} ({{gender}} - {{participant.tshirtSize}}){{#if @first}}
-                                <br/>{{/if}}
-                            {{/each}}
+                            {{/with}}
                         </td>
+                        <td>{{purposeOfTransfer}}</td>
                     </tr>
                 {{/each}}
                 </tbody>
             </table>
         </div>
     </div>
+    <!-- Invoices -->
+    <div role="tabpanel" class="tab-pane" id="invoices">
+        <div class="container">
+            <div class="registration-margin"></div>
+            <div id="invoices_out"></div>
+
+            {{#each confirmedInvoices}}
+                  <h3>
+                    {{event.title}}
+                    {{#if hasFullyPaid}}
+                      ✅
+                    {{else}}
+                      ❗️
+                    {{/if}}
+                  </h3>
+
+                  <h4>{{__ 'INVOICE_OVERVIEW'}}</h4>
+
+                  <table class="table table-striped text-left" style="min-width: 450px; width: 50%">
+                      <tr>
+                          <td>{{__ 'RECIPIENT'}}</td>
+                          <td>BreakOut e.V.</td>
+                      </tr>
+                      <tr>
+                        <td>IBAN</td>
+                        <td>DE85 7002 2200 0020 2418 37</td>
```

Comment:
Should this be public?

Reply:
It's sent by email. And is hardcoded in mail service as well. It's not exactly private now